### ISSUE TITLE
[Bug]: Incompatible with Carpet

### ISSUE THREAD

**Issue by Greg-J** [LarsMans64/InfinityButtons#32]
### Bug

1. What happened?
I have an obscene number of mods that all work fine together, but when I add this mod I lose the ability to use the /player command.

### What version of the mod are you playing?

Fabric

### What version of Minecraft are you playing?

1.19.2

### Latest version

- [X] I am running the latest version of the mod

### latest.log

https://mclo.gs/xjXt0yJ

**Comment by curryducker** [comment]
Infinity Buttons isn't even present?

**Comment by Greg-J** [comment]
> Infinity Buttons isn't even present?

I had to remove it because it was preventing Carpet from using the /player command.

**Comment by LarsMans64** [comment]
What exactly happens? Is the command just not there or does it give an error?

**Comment by Greg-J** [comment]
> What exactly happens? Is the command just not there or does it give an error?

The response in chat is "An unexpected error occurred trying to execute that command" and the tooltip when you hover over it reads "Payload may not be larger than 1048576".

Here is the full log of booting up with the mod, running the command, and shutting down: https://mclo.gs/e6qhE0h

**Comment by LarsMans64** [comment]
Could you try it with only infinity buttons and carpet installed? by the way i think you've got more problems than only this, 99% of your logs is errors

**Comment by curryducker** [comment]
yes and 75% of those errors are still on us because of the bad implementation of compat on IB fabric. anyway, besides that issue that shouldn't impact gameplay i see nothing infinity buttons related in the logs that can cause the issue.

**Comment by Greg-J** [comment]
> Could you try it with only infinity buttons and carpet installed? by the way i think you've got more problems than only this, 99% of your logs is errors

217 Errors, and they're all issues parsing recipes, advancements, and loot tables which is suuuuuper common with mods. I don't have any issues in-game or crashes. The only issue I have is this one where IB prevents the use of the /player command.

**Comment by dhouck** [comment]
I can use `/player` just fine with IB, although I’m on 1.20.1; a log with just the two mods would probably still help.  This may be a three-mod interaction issue not just an incompatibility between the two.
